migrations/20220505003315-is-frozen.js

```diff
@@ -0,0 +1,45 @@
+'use strict';
+
+module.exports = {
+  async up(queryInterface, Sequelize) {
+    /**
+     * Add altering commands here.
+     *
+     * Example:
+     * await queryInterface.createTable('users', { id: Sequelize.INTEGER });
+     */
+    queryInterface
```

Comment:
Need to return the promise for sequelize to properly await. We tend to prefer using `async`/`await`